docs/_includes/helpers/mobile-embed.html

```diff
@@ -0,0 +1,39 @@
+
+<div class="mobile-embed">
```

Comment:
remove this extra blank line?
```suggestion
<div class="mobile-embed">
```

Reply:
Yep good catch 👍